### PR TITLE
DDF-2824 Update Caching Federation Strategy docs

### PR DESCRIPTION
**Pull request by emanns95**
#### What does this PR do?
Updates Caching Federation Strategy configuration documentation is accurately define the default value for cache everything. 
https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/sortedFederationStrategy.xml#L64

#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@ahoffer 
@peterhuffer 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams/docs

#### Choose 2 committers to review/merge the PR. 
@pklinef
@ricklarsen - Documentation

#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc [hunk 1]
```diff
@@ -37,11 +37,11 @@
...
 
-|Cache Everything
-|cachingEverything
-|Boolean
-|Cache all results unless configured as native
-|false
+|Query Result Cache Strategy
+|cacheStrategy
+|String
+|Strategy for caching query results. Valid entries are ALL, FEDERATED, and NONE.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc]: We should add the second sentence to the metatype.xml file that this table maps too

Comment (emanns95) [reply distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc]: I'll leave it out if that's okay so I don't have to pull down this branch 😂

**Comment by emanns95** [comment]
build now

File: distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc]: 👍

**Pull request merged**